app/config/routing.yml

```diff
@@ -7,6 +7,11 @@
 app:
     resource: @AppBundle/Controller/
     type:     annotation
+    prefix:   /{_locale}
+    requirements: 
+        _locale: en|fr|de|es|cs|ru|uk| 
+    defaults:
+        _locale: en
```

Comment:
I think we should use `_locale: "%locale%"` in order to use `locale` parameter from `parameters.yml`